### ISSUE TITLE
Cannot tab to 'copy to clipboard'

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#3576]
On the project detail page, we have a link where the user can copy the pip install command to their clipboard:

![screenshot from 2018-04-07 07-48-51](https://user-images.githubusercontent.com/3323703/38452258-4f46712e-3a38-11e8-9cb7-f23d7391a306.png)

The code for this is:

```html
<a class="-js-copy-pip-command tooltipped tooltipped-s" data-clipboard-target="#pip-command" aria-label="Copy to clipboard" data-original-label="Copy to clipboard">
  <i class="fa fa-copy" aria-hidden="true"></i>
  <span class="sr-only">Copy PIP instructions<span>
</a>
```

We need to ensure that a user using a keyboard *only* can access this functionality.

Right now we are using a `<a>` with no `href` for this element.  When tabbing through the page using a keyboard, this "empty" `<a>` is *ignored*.  Instead we should use a `<button>` element, which will be recognised in the tabindex.

Making this change _may_ mean that we have to update the associated JavaScript.

---

Note: when making this change, we should also update the 'copy to clipboard' links on `/templates/manage/release.html`:

- https://github.com/pypa/warehouse/blob/master/warehouse/templates/manage/release.html#L152
- https://github.com/pypa/warehouse/blob/master/warehouse/templates/manage/release.html#L161
- https://github.com/pypa/warehouse/blob/master/warehouse/templates/manage/release.html#L170
